src/character/CharacterHandler.cpp

```diff
@@ -392,7 +412,7 @@ bool CharacterHandler::validate_options(const Character& character, std::uint32_
 		}
 	}
 
-	// facial features (horns, markings, tusks, piercings, hair) validation
+	// facial features (horns, markings, tusks, piercings, beard) validation

```

Comment:
This wasn't a mistake, please undo. ;)
```suggestion
	// facial features (horns, markings, tusks, piercings, hair) validation
```

Reply:
I thought it was since it was nested inside 'facial features', I'll undo this change again.